src/elements/content-sidebar/additional-tabs/__tests__/AdditionalTab.test.js

```diff
@@ -69,4 +69,23 @@ describe('elements/content-sidebar/additional-tabs/AdditionalTab', () => {
 
         expect(wrapper).toMatchSnapshot();
     });
+
+    test('should render the FTUX tooltip when ftuxTooltipData is present', () => {
+        const mockSrc = 'https://foo.com/image';
+        const props = {
+            title: 'test title',
+            iconUrl: mockSrc,
+            isLoading: false,
+            id: 4,
+            callback: () => {},
+            ftuxTooltipData: {
+                targetingApi: () => {},
+                text: 'ftux text',
+            },
+        };
+
+        const wrapper = getWrapper(props);
+
+        expect(wrapper).toMatchSnapshot();
```

Comment:
Snapshots tend to be pretty brittle, can we do an explicit assertion here on what we expect?